CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## [Unreleased]
```

Comment:
1. See [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).
2. To add a new changelog entry: `npm run newChange -w packages/amazonq` or `npm run newChange -w packages/toolkit`